src/main/java/cz/student/zvedamrance/App.java

```diff
@@ -1,4 +1,30 @@
 package cz.student.zvedamrance;
 
+import cz.common.DbUtil;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
 public class App {
+    public static void main(String[] args) {
+        try {
+            Connection connection = DbUtil.getConnection();
+            System.out.println("Connected to database");
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        UserDao userDao = new UserDao();
```

Comment:
Pouzivat defaultne nemenne promenne, pokud neni potreba menit odkaz do pameti v prubehu prace programu.

Duvod
Používání final proměnných v Javě je dobré, protože zajišťuje neměnnost hodnoty po její inicializaci, což zvyšuje bezpečnost a predikovatelnost kódu. Tím se eliminuje riziko neúmyslných změn hodnoty, což usnadňuje debugging a údržbu kódu. Navíc final proměnné mohou pomoci optimalizátorům kódu při kompilaci nebo běhu programu, protože zaručují, že hodnota nebude měněna, což může vést k lepšímu výkonu a efektivnější správě paměti.

```suggestion
final UserDao userDao = new UserDao();
```

Reply:
opraveno